code/game/machinery/cloning.dm

```diff
@@ -109,7 +111,27 @@ GLOBAL_VAR_INIT(clones, 0)
 	. = ..()
 	. += "The write-protect tab is set to [read_only ? "protected" : "unprotected"]."
 
+// Biomass
 
+/obj/machinery/clonepod/attackby(obj/item/W, mob/user, params)
+	var/old_biomass = biomass // saves biomas variable
+	if(istype(W, /obj/item))
```

Comment:
You dont need to check if its an item since you already check if its a specific item later down the line

Reply:
I know that but I personally just like the code better and it feels easier to understand when you have EVERY check put under one if(istype so that other people can see oh that's the overall check, so if I want to add something other than meat I put it under here.